axiom/tests/test_txn.c

```diff
@@ -7545,7 +7570,7 @@ static void test_get_current_trace_id(void) {
   txn_id = nr_txn_get_guid(txn);
   trace_id = nr_txn_get_current_trace_id(txn);
   tlib_fail_if_null("txn id", txn_id);
-  tlib_pass_if_str_equal("txn_id == trace_id", txn_id, trace_id);
+  tlib_pass_if_str_equal("padded txn_id == trace_id", txn_id, trace_id);
```

Comment:
Why has this test message been modified? This test should have two versions: one when txn is started with pad_trace_id set to false and another when pad_trace_id is set to true.

Reply:
f71fca4a2ba3fdb2f17d5816ef3f9abc63f3cdd7 fixes this unit test by providing better explanations of what is tested and two cases are tested: one with trace_id being padded and one with trace_id not being padded.